tests/test_CQCToFile.py

```diff
@@ -0,0 +1,185 @@
+# Tests for the CQCToFile class
+
+# To get a temporary directory start the function with
+#
+# filename = os.path.join(str(tmpdir), 'CQC_File')
+#
+# with CQCToFile(filename=filename) as cqc:
+
+from cqc.pythonLib import CQCToFile, qubit
+import os
+from cqc.cqcHeader import CQC_TP_HELLO
+
+
+def test_name(tmpdir):
+
+    filename = os.path.join(str(tmpdir), 'CQC_File')
+
+    with CQCToFile(filename=filename) as cqc:
+        assert cqc.name == 'CQCToFile'
+
+
+def test_sendSimple(tmpdir):
+
+    filename = os.path.join(str(tmpdir), 'CQC_File')
+
+    with CQCToFile(filename=filename) as cqc:
+         
+        cqc.sendSimple(CQC_TP_HELLO)
+
+        with open(filename) as f:
+            contents = f.read()
+            print(contents)
+            assert contents[6:10] == "\\x00"
```

Comment:
It would be better if you instanciate the headers here (from `cqcHeader`) and check the command w.r.t the variable in the same file. Otherwise, if we cange the headers we also need to change these tests.

Reply:
Maybe with these tests it would be better to just use and read the binary files. In that case it's much easier to extract the numbers. With the text file it would be a bit more complicated to translate "\\x00" to a number of to have to `struct.pack()` the number, convert it to text and then compare. What do you think?

Reply:
@lgvacanti I'm not sure I fully understand. I think it would still be good to instanciate the headers using `cqcHeader`